python/test/unit/language/test_core.py

```diff
@@ -1216,3 +1216,36 @@ def _kernel(dst):
     _kernel[(1,)](dst=dst, num_warps=1)
     _kernel[(1,)](dst=dst, num_warps=2)
     _kernel[(1,)](dst=dst, num_warps=4)
+
+
+@pytest.mark.parametrize("dtype_str", dtypes)
+def test_broadcast(dtype_str):
```

Comment:
Do you think we could have this test be a special case of https://github.com/openai/triton/blob/master/python/test/unit/language/test_core.py#L383, since it tests broadcasting over different axes individually

Reply:
Sure, that's a good idea. I just combined two test cases together.